### DISCUSSION TITLE
Open SDG for non-SDG metrics: indicators associated with multiple goals and targets

### DISCUSSION THREAD

**Discussion by EFT-Defra** [open-sdg/open-sdg#1429] [Ideas]
Hi @brockfanning and @LucyGwilliamAdmin,

I'm part of a team at Defra that's trying to adapt OpenSDG to report our [Outcome indicator framework for the 25 Year Environment Plan (OIF)](https://www.gov.uk/government/publications/25-year-environment-plan-progress-reports). See our site [here](https://github.com/Defra-Data-Science-Centre-of-Excellence/OIF-Dashboard-Site). We've had some success following the "translations" approach suggested in #293 and documented [here](https://open-sdg.readthedocs.io/en/latest/translation/#overriding-and-extending-translation-data). However, one major difference between OIF and SDG is that OIF indicators can be associated with multiple goals and targets. 

For example, our indicator "A1: Emissions for five key air pollutants in England, 1998 to 2018" (our indicators are currently encoded by theme, instead of by goal and target, but that's a separate matter...) is primarily associated with goal 1, target 1 but secondarily associated with goal 3, target 5.

It doesn't look like associating an indicator with a second goal/target is possible in SDG but I'm pretty new to Jekyll/Ruby and I might be missing something obvious. Any ideas?

**Comment by brockfanning** [comment]
@EFT-Defra, your hunch is right that Open SDG doesn't support that kind of data structure out of the box. There are probably ways to get closer to that, such as through new metadata fields and overriding layouts. Also I should mention that this issue happens in the SDGs as well. There are some "multi-purpose" indicators that show up in multiple goals/targets. And example is 1.5.1, 11.5.1, and 13.1.1. In the SDGs, this happens rarely enough that so far we haven't added any special handling for the issue, and implementations just need to duplicate the indicators by having separate data/metadata for each instance of the indicator.

**Comment by EFT-Defra** [comment]
Okay, so even though it isn't a pressing issue for SDG, you'd be interested if we got something working for OIF?

As you suggest, I was thinking of adding some secondary metadata fields. Will I need to look at the python scripts or the Jekyll plugin too?

**Comment by brockfanning** [comment]
Yes, depending on the solution it may involve some Ruby in jekyll-open-sdg-plugins, and possibly some Python in sdg-build too.

Do you picture these indicators as having multiple different URLs? Or do you picture them as one URL that's linked from multiple goal pages?
